### PR TITLE
Turn on xplat tests

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by HaoK** [comment]
@divega This turns on the xplat test runs, unfortunately the current shared code has the attributes, so even though these should pass for the in memory providers, they are picking up the Skips still.  Filed an issue for this separate from the EF specific local db one...

https://github.com/aspnet/Identity/issues/673

**Comment by HaoK** [comment]
Updated removing the attributes and using a manual skip method check which does the same checks, this allows the non SQL providers to run on xplat now.

File: test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class CustomDbContext<TKey> : DbContext where TKey : IEquatable<TKey>
 
         [TestPriority(-1000)]
         [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)][OSSkipCondition(OperatingSystems.Linux)][OSSkipCondition(OperatingSystems.MacOSX)]
```
Comment (HaoK) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: These are one off, standalone non shared tests, so they can continue to use the skip attributes

Comment (divega) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: Do you actually mean we will never want to run them on Linux and Mac? Or is it just a temporary limitation?

Comment (divega) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: Nit: split the lines to have just one attribute per line.

Comment (HaoK) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: Basically until https://github.com/aspnet/Identity/issues/670 is fixed

**Comment by divega** [comment]
:shipit: but the skip conditions in those "one off, standalone, non-shared" tests smell like regular engineering debt :smile: If that is the case, please make sure we have a bug tracking it.

File: test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs (hunk 1 above)
Comment (HaoK) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: Or we forever if decide this one test project can be windows only :)

Comment (divega) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs]: > Or forever if we decide this one test project can be windows only :)

Not sure why we would decide such thing, but we can talk :smile:

**Comment by HaoK** [comment]
a66f50c1ba2dec01b0ca001dc22767d77ece29f2

**Pull request closed without merging**